assets/csidriveroperators/vsphere/08_deployment.yaml

```diff
@@ -50,6 +50,10 @@ spec:
           requests:
             memory: 50Mi
             cpu: 10m
+        volumeMounts:
+        - name: trusted-ca-bundle
+          mountPath: /etc/pki/ca-trust/extracted/pem
+          readOnly: true
```

Comment:
nit - configmaps are already readonly.

Reply:
It's copied from our docs :-). I removed it.